From: Timofei Fedotov <sovtouch@altlinux.org>
Date: Wed, 3 Jun 2026 13:36:10 +0300
Subject: fix: global-voice-lock-mode initialization for Emacs 29

Replace custom-initialize-delay with custom-initialize-default in
define-globalized-minor-mode for global-voice-lock-mode in voice-setup.el.

custom-initialize-delay causes void-function error on Emacs 29 because
it attempts to call global-voice-lock-mode during custom variable
initialization before the function is defined by define-globalized-minor-mode.
---
 lisp/voice-setup.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lisp/voice-setup.el b/lisp/voice-setup.el
index 31896cc..5f29cd4 100644
--- a/lisp/voice-setup.el
+++ b/lisp/voice-setup.el
@@ -516,7 +516,7 @@ command \\[customize-variable] on <personality>-settings. "
 
 (define-globalized-minor-mode global-voice-lock-mode
   voice-lock-mode turn-on-voice-lock
-  :initialize 'custom-initialize-delay
+  :initialize 'custom-initialize-default
   :init-value (not (or noninteractive emacs-basic-display))
   :group 'voice-lock
   :version "24.1")
